plugins/woocommerce-admin/client/products/sections/product-details-section.tsx

```diff
@@ -24,9 +24,9 @@ import { recordEvent } from '@woocommerce/tracks';
  * Internal dependencies
  */
 import './product-details-section.scss';
+import { getCheckboxProps, getTextControlProps } from './utils';
```

Comment:
Just curious why this changed... was is auto-changed by prettier/linter, or did you manually change it? Do we have guidelines around import ordering?

Reply:
Interested... did not know we had a guideline around this!

Reply:
A linting rule would be excellent in the future.